lib/ResourceSet.h

```diff
@@ -19,169 +20,168 @@ using TResourceCap = int64_t; //to avoid overflow when adding integers. Signed v
 class JsonNode;
 class JsonSerializeFormat;
 
-namespace Res
+class ResourceSet;
+
+enum class ECurrencyID : int8_t

```

Comment:
Please rename it to `EResourceID`, which is the name normally used for resources. We don't mine any currency here.

Reply:
@IvanSavenko @DjWarmonger  maybe EGameResID? It is less confusing.

Reply:
Either will do for me

Reply:
Renamed to GameResID/EGameResID.

Reply:
Will push after all topics will be agreed.